src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+const errorHandler = ((err, req, res, next) => {
+    console.log('Error', err);
```

Comment:
Check whether all the headers are sent or not if not then throw error

Reply:
Done